packages/bierzo-wallet/src/logic/transactions/types/BwSendTransaction/index.tsx

```diff
@@ -38,7 +37,7 @@ export class BwSendParser extends BwParser<ProcessedSendTransaction> {
       `"${original.amount.tokenTicker}"`,
     ];
     const date = [`"${tx.time.toISOString()}"`];
-    const status = [`"${tx.received}"`, `"${original.memo}"`];
+    const status = [`"${original.memo}"`];
 
     const txRow = [...parties, ...payment, ...date, ...status];
```

Comment:
I'm not convinced all these arrays/destructuring make things any clearer.

Reply:
I fully agree. However, the whole CSV idea needs a big rework since we want to support different kind of transactions in the same table. I'm resolving this as unrelated for now.